solr/core/src/java/org/apache/solr/core/FileSystemConfigSetService.java

```diff
@@ -62,7 +62,12 @@ public String configSetName(CoreDescriptor cd) {
   @Override
   public boolean checkConfigExists(String configName) throws IOException {
     Path configSetDirectory = configSetBase.resolve(configName);
-    return Files.isDirectory(configSetDirectory);
+    if (Files.isDirectory(configSetDirectory)) {
```

Comment:
Why the isDirectory check; why not just check for solrconfig.xml?

Reply:
Fixed